api/src/pcapi/core/finance/api.py

```diff
@@ -96,14 +97,18 @@
 def get_pricing_ordering_date(
     booking: bookings_models.Booking | educational_models.CollectiveBooking,
 ) -> datetime.datetime:
-    stock: offers_models.Stock | educational_models.CollectiveStock = (
-        booking.stock if isinstance(booking, bookings_models.Booking) else booking.collectiveStock
-    )
+    if isinstance(booking, bookings_models.Booking):
+        eventDatetime = booking.stock.beginningDatetime
+    else:
+        if feature.FeatureToggle.USE_END_DATE_FOR_COLLECTIVE_PRICING.is_active():
+            eventDatetime = booking.collectiveStock.endDatetime
+        else:
+            eventDatetime = booking.collectiveStock.beginningDatetime
```

Comment:
Pour éviter un niveau de profonder 😄 
```suggestion
    if isinstance(booking, bookings_models.Booking):
        eventDatetime = booking.stock.beginningDatetime
    elif feature.FeatureToggle.USE_END_DATE_FOR_COLLECTIVE_PRICING.is_active():
        eventDatetime = booking.collectiveStock.endDatetime
    else:
        eventDatetime = booking.collectiveStock.beginningDatetime
```

Reply:
Je préfère justement la version indentée pour bien différencier le collectif vs l'individuel.